lib/tests/common.go

```diff
@@ -0,0 +1,144 @@
+package test
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/appc/docker2aci/lib/internal/typesV2"
+)
+
+type Layer map[*tar.Header][]byte
```

Comment:
You are right about the layer hashes, it is too late to do reviews ;-) Let's leave it like this then.

If I'll come up with a better idea, I'll do a follow-up PR.

LGTM